### PR TITLE
Modified some things from Back for good functionality.

### PR DESCRIPTION
**Pull request by martinalturria**
- Fix enum in models    
- Convert ENUM to String in Enterprise, User Trip models
- modularization of routes by table
- Add Handlers for Trips.
- Modify Trip Controllers for Logica
- Fix data type in ratings

### CODE REVIEW

**Review by brunofoster03 — DISMISSED** [review]
Great👍

File: src/handlers/Entrepise/entrepisesHandler.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
+const getEnterpriseController = require("../../controllers/Enterprise/getEnterpriseController.js");
+const getEnterprisesController = require("../../controllers/Enterprise/getEntrepisesController.js");
+const postEnterpriseController = require("../../controllers/Enterprise/postEnterpriseController.js");
+const putEnterpriseController = require("../../controllers/Enterprise/putEnterpriseController.js");
+const statusEnterpriseController = require("../../controllers/Enterprise/statusEnterpriseController.js");
+
+const getEntrepisesHandler = async (req, res) => {
```
Comment (manduinca) [inline src/handlers/Entrepise/entrepisesHandler.js]: Typo.

File: src/handlers/Entrepise/entrepisesHandler.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    try {
+        const entrepises = await getEnterprisesController();
+
+        res.status(200).json(entrepises);
+    } catch (error) {
+        return res.status(400).json({ error: error.message });
+    }
+};
+
+const getEntrepiseHandler = async (req, res) => {
```
Comment (manduinca) [inline src/handlers/Entrepise/entrepisesHandler.js]: Another typo.

File: src/handlers/Entrepise/entrepisesHandler.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+            address
+        );
+
+        return res.status(200).json(entrepiseUpdate);
+    } catch (error) {
+        return res.status(400).json({ error: error.message });
+    }
+};
+
+const statusEntrepiseHandler = async (req, res) => {
```
Comment (manduinca) [inline src/handlers/Entrepise/entrepisesHandler.js]: Typo.

File: src/handlers/Entrepise/entrepisesHandler.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    } catch (error) {
+        return res.status(400).json({ error: error.message });
+    }
+};
+
+const putEntrepiseHandler = async (req, res) => {
+    const { ruc, newRuc, name, address } = req.body;
+
+    try {
+        const entrepiseUpdate = await putEnterpriseController(
```
Comment (manduinca) [inline src/handlers/Entrepise/entrepisesHandler.js]: Also, here.

File: src/handlers/Entrepise/entrepisesHandler.js [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+        const entrepise = await postEnterpriseController(ruc, name, address);
+
+        return res.status(201).json(entrepise);
+    } catch (error) {
+        return res.status(400).json({ error: error.message });
+    }
+};
+
+const putEntrepiseHandler = async (req, res) => {
```
Comment (manduinca) [inline src/handlers/Entrepise/entrepisesHandler.js]: Typo.

File: src/handlers/Entrepise/entrepisesHandler.js [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+        const entrepise = await getEnterpriseController(ruc);
+
+        return res.status(200).json(entrepise);
+    } catch (error) {
+        return res.status(400).json({ error: error.message });
+    }
+};
+
+const postEntrepiseHandler = async (req, res) => {
```
Comment (manduinca) [inline src/handlers/Entrepise/entrepisesHandler.js]: Typo.

File: src/routes/index.js [hunk 7]
```diff
@@ -1,42 +1,23 @@
...
-router.get('/trip/get', getTrip)
-router.put('/trip/update', putTrip)
-router.put('/trip/status', statusTrip)
-
-// Exports router
+router.post('/auth', authHandler)
+
+router.use('/user', userRouters)
+
+router.use("/entrepise", entrepiseRouters)
```
Comment (manduinca) [inline src/routes/index.js]: Typos.

**Review by manduinca — CHANGES_REQUESTED** [review]
Plase check it, there are some typos.

**Comment by martinalturria** [comment]
requested fix typo.

**Review by manduinca — APPROVED** [review]
LGTM!

**Review by roquemolina — APPROVED** [review]
Correct

**Pull request merged**